### PR TITLE
updated values for rabbitmq secret

### PR DESCRIPTION
**Pull request by mpigram**
When the rabbitmq-service is created and deployed into the icap-adaptation namespace a secret is created called `rabbitmq-service-default-user`. This has the randomly generated username and password for the services to access Rabbitmq.

Instead of manually creating a secret, these changes should reference the secret that is created on deployment and take away the manual process.

### CODE REVIEW

**Comment by gargath** [comment]
This should work,  but only as long as the `RabbitmqCluster` lives in the same namespace as the adaptation service components (secrets are not accessible cross-namespaces).

Unfortunately Rancher manages apps in such a way that each Chart has to live within its own namespace.
So, to make this work in conjunction with Rancher, the `RabbitmqCluster` resource would need to move into the adaptation chart. Then this will come together nicely in both AKS and Rancher-based environments.

Any concerns with that approach?

**Comment by mpigram** [comment]
> This should work, but only as long as the `RabbitmqCluster` lives in the same namespace as the adaptation service components (secrets are not accessible cross-namespaces).
> 
> Unfortunately Rancher manages apps in such a way that each Chart has to live within its own namespace.
> So, to make this work in conjunction with Rancher, the `RabbitmqCluster` resource would need to move into the adaptation chart. Then this will come together nicely in both AKS and Rancher-based environments.
> 
> Any concerns with that approach?

Just so I'm on the same page.

We have `rabbitmq-service-server` which currently lives in the namespace `icap-adaptation` - then we have the `rabbitmq-cluster-operator` that lives in it's own namespace `rabbitmq-system'.

You want all rabbitmq services to live inside 'icap-adaptation'?

**Review by gargath — APPROVED** [review]
lgtm

File: adaptation/values.yaml [hunk 1]
```diff
@@ -1,4 +1,10 @@
 environment:
+rabbitmqService:
```
Comment (BigPeeg) [inline adaptation/values.yaml:2]: Nothing is using these values

**Review by BigPeeg — APPROVED** [review]

File: adaptation/values.yaml (hunk 1 above)
Comment (mpigram) [reply adaptation/values.yaml:2]: the rabbitmq-server-service is? I've just copied the yaml across and added the values. Or am I missing something?

**Pull request merged**